composer.json

```diff
@@ -30,6 +30,14 @@
     {
       "name": "Toon Verwerft",
       "email": "toon.verwerft@phpro.be"
+    },
+    {
```

Comment:
To make sure that every author gets credit for their code, we use this [contributors list](https://github.com/phpro/soap-client-sf2-bridge/graphs/contributors) as the single source of truth. This way we can keep the `composer.json` file clean and we don't have to manage the contributors list manually. Can you remove these lines? Thanks!